druntime/src/core/int128.d

```diff
@@ -18,8 +18,23 @@ alias I = long;
 alias U = ulong;
 enum Ubits = uint(U.sizeof * 8);
 
-version (X86_64) private enum Cent_alignment = 16;
-else             private enum Cent_alignment = (size_t.sizeof * 2);
+version (DigitalMars)
+{
+    /* The alignment should follow target.stackAlign(),
+     * which is `isXmmSupported() ? 16 : (is64bit ? 8 : 4)
+     */
+    version (D_SIMD)
+        private enum Cent_alignment = 16;
+    else version (X86_64)
+        private enum Cent_alignment = 4;
```

Comment:
Looks like these last two cases are interchanged - this should be `8` and the else branch below should be `4`. However, doesn't `X86_64` imply `D_SIMD` anyway? X86-64 has SSE2 support in its base spec.

Reply:
The layout exactly matches what the dmd compiler does.

Reply:
The comment above:

> `is64bit ? 8 : 4`

vs.

> ```
> version (X86_64)
>     private enum Cent_alignment = 4;
> else
>     private enum Cent_alignment = 8;

looks to be in contradiction to me.

Reply:
I see it now. Oops. good catch